src/test/java/com/amazon/ion/impl/bin/FlexIntTest.kt

```diff
@@ -32,6 +32,7 @@ class FlexIntTest {
         "             1048576, 00001000 00000000 00000000 00000001",
         "           134217727, 11111000 11111111 11111111 01111111",
         "           134217728, 00010000 00000000 00000000 00000000 00000001",
+        "    ${Int.MAX_VALUE}, 11110000 11111111 11111111 11111111 00001111",
```

Comment:
🗺️ This got added just because I was asking myself what is the FlexInt representation of `Int.MAX_VALUE`, so now it's here for reference.